atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPoolImpl.java

```diff
@@ -168,7 +179,9 @@ private CassandraClientPoolImpl(
                 .build());
         this.blacklist = blacklist;
         this.exceptionHandler = new CassandraRequestExceptionHandler(
-                this::getMaxRetriesPerHost, this::getMaxTriesTotal, blacklist);
+                () -> runtimeConfig.get().numberOfRetriesOnSameHost(),
```

Comment:
We could just pass the supplier to the `ExceptionHAndler` now.

Reply:
The supplier here is of CKVSRuntimeConfig. This is how we make a supplier of Integer.

Reply:
Yeah, I meant give it the `Supplier<Config>` so it can just access what it needs.  Not important right now, we can do it if it starts reading more config parameters.